### DISCUSSION TITLE
Multiple ports

### DISCUSSION THREAD

**Discussion by kamnev999** [postalserver/postal#3213] [Help with using Postal]
Hello.
It became necessary to use several ports.

The problem is as follows:
1. I cannot use the standard outgoing ports (port 25)
2. I use port 588, but incoming mail does not work on it.

smtp_server:
 default_bind_address: "::"
 tls_enabled: true
 default_port: 588

No matter how I try to add 2 ports 588 and 25, only 1 works.

How can I solve my problem?
Is it possible to set up incoming mail on port 588?

**Comment by willpower232** [comment]
Postal can only listen on one port and 25 is the most recommended choice since Postal only supports explicit SSL with STARTTLS.

You can use iptables to listen on multiple ports at the same time, essentially forwarding packets from one port to the one Postal is actually on. Have a look at https://github.com/postalserver/postal/issues/91 and other issues

If you don't have port 25 open, you probably won't receive any incoming email or message bounces and receiving servers may not trust your Postal since they can't confirm it is a mail server.

**Reply by Dieg12** [reply]
I'm facing a use case where I want my clients to use SSL for more safety but some of my clients do not support it. If i configure my server with tls_enabled: true , will my old clients still be able to not use SSL ?

**Reply by willpower232** [reply]
> Postal only supports explicit SSL with STARTTLS

if your clients don't support STARTTLS then they can still use Postal, they just don't opt in to it.

All SMTP connections to Postal are insecure by default and then upgraded to SSL with STARTTLS.

**Reply by Dieg12** [reply]
Thank you very much for the quick reply, I feel like this should be noted in the documentation tho.
